### DISCUSSION TITLE
Multivariable regression example

### DISCUSSION THREAD

**Discussion by BldgICT** [jeshraghian/snntorch#201] [Q&A] [answered]
Hi, I'm studying spiking neural networks recently.
Big thanks to snnTorch library developers.
While following tutorials, I understood the single variable regression model.
By the way, is there any multivariable regression example code?
For example, using the famous house price prediction dataset...
Is there anyone who can help?

**Comment by jeshraghian** [comment, marked as answer]
[Part II](https://snntorch.readthedocs.io/en/latest/tutorials/tutorial_regression_2.html) in the regression tutorials is an example of multivariable regression - 784 variables of pixels to be exact.

Update the architecture and loss function to suit your needs.

**Comment by BldgICT** [comment]
Thank you for answer.
I am writing code while referring to Tutorial part 2.
However, when evaluating the test dataset, can't I use mean squared error instead of accuracy as the evaluation metric?
When the number of output neurons increases, accuracy does not seem to be an appropriate metric.

**Reply by jeshraghian** [reply]
You should use whatever evaluation metric that is appropriate for your dataset. 
I can't really help without understanding why you're increasing the number of output neurons for regression.
